mitosheet/css/toolbar.css

```diff
@@ -135,8 +135,8 @@
   font-family: Roboto;
 }
 
-.mito-toolbar-button-container-enabled:hover {
-  background-color: var(--mito-highlight-light);
+.mito-toolbar-button-container-enabled:hover .mito-toolbar-button-icon-container {
+  background-color: var(--mito-light-gray);
```

Comment:
This should be `--mito-very-light-gray`. It's darker than the mockup.

Reply:
Oh yeah I think `--mito-very-light-gray` is actually the current background color of the toolbar. Maybe I can make an in-between gray?

Reply:
I think it's actually ok to leave as is for now.